s3auth-relay/src/main/java/com/s3auth/relay/HttpFacade.java

```diff
@@ -256,6 +259,9 @@ public void run() {
             } catch (final InterruptedException ex) {
```

Comment:
maybe we can use `VerboseRunnable` instead, see http://log.jcabi.com/threads-VerboseRunnable.html